app/src/main/java/de/westnordost/streetcomplete/quests/handrail/AddHandrail.kt

```diff
@@ -0,0 +1,48 @@
+package de.westnordost.streetcomplete.quests.handrail
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+/**
+ * Only ask about whether there exists a handrail, not about the location (left, right and/or
+ * middle), because the primary use case here is accessibility of the path (not 3d rendering, etc),
+ * which does not depend on the location of the handrail.
+ * The details would only slow down the answering in a survey by requiring more thought and a more
+ * complex GUI (interpreting the direction of the OSM way for example).
+ *
+ * Github Issue for this quest:
+ * https://github.com/westnordost/StreetComplete/issues/1390
+ */
+class AddHandrail(overpassServer: OverpassMapDataDao) : SimpleOverpassQuestType<Boolean>(overpassServer) {
+    // Do not include nodes and relations, even though these exist with the right tags, because
+    // according to the wiki page for `highway=steps` [2] it can only be applied to ways. It also
+    // does not make much sense for other types of elements and it is more likely to be a
+    // Do not exclude elements tagged with `handrail:left`, `handrail:center` or `handrail:right`
+    // because according to the wiki page for `handrail=*` [1], the `handrail` tag should
+    // always have just `yes` or
+    // `no` as a value to indicate the existence of a handrail somewhere. If there is a mismatch of
+    // the tags afterwards, that can be recognized using one of the various
```

Comment:
In other situations we skipped the quest in such a situation. So I would exclude `handrail:*` here as well. 
Surveyor time is valuable...

Reply:
So you would assume that if only `handrail:*` is tagged, the survey was valid and should not be repeated by Streetcomplete, do I understand that correctly?

Reply:
Yes